pyproject.toml

```diff
@@ -55,6 +55,10 @@ docs = [
     "mkdocs-macros-plugin",
 ]
 
+[tool.setuptools.packages.find]
+include = ["pynxtools_xps*"]
+exclude = ["docs*", "examples*", "tests*"]
```

Comment:
Excluding all these packages is not really necessary if an include is present.

Reply:
You are right, I remove the exlucde line.